sechub-pds/src/main/java/com/mercedesbenz/sechub/pds/IntegrationTestPDSRestController.java

```diff
@@ -60,8 +64,13 @@ public void isAlive() {
     @RequestMapping(path = PDSAPIConstants.API_ANONYMOUS + "integrationtest/shutdown", method = RequestMethod.GET, produces = {
             MediaType.APPLICATION_JSON_VALUE })
     public void shutdownServer() {
-        LOG.info("Integration test server shutdown initiated by closing context...");
+        LOG.info("Integration test server shutdown wished...");
```

Comment:
Suggest to use a more common wording:
```suggestion
        LOG.info("Integration test server shutdown requested...");
```